### PR TITLE
Term entry isinf

### LINKED ISSUE #967: [Term Entry] C++ Math Functions: `isinf()`
### Reason/inspiration (optional)

Under `cpp/concepts/math-functions/terms/`, create a new term entry for the `isinf()` function.

- In the `terms/` folder, make a new folder called `isinf`
- Inside the `isinf` folder, create a new markdown file called `isinf.md`
- Inside the file, be sure to include a "Syntax" section with a \```pseudo code snippet.
- Let's include an "Example" section with a \```cpp code snippet.
- Lastly, let's include a "Codebyte Example" with a runnable \```codebyte/cpp

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] I agree to follow Codecademy Doc's Code of Conduct

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by ddiegow**
### Description

Create isinf() entry. Closes #967 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by SSwiniarski** [comment]
@ddiegow Just FYI, you should make new branches off your main branch, not off of prior PR branches. This isn't a big deal this time, because your remquo PR was already merged, but it could cause problems if you branch off a PR branch that's still being reviewed. You don't have to change anything in this PR because of this, but keep it in mind for future PRs.

File: content/cpp/concepts/math-functions/terms/isinf/isinf.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+---
+Title: 'isinf()'
+Description: 'Returns a value telling whether the argument is an infinite value.'
```
Comment (SSwiniarski) [inline content/cpp/concepts/math-functions/terms/isinf/isinf.md:3]:
```suggestion
Description: 'Returns a value indicating whether the argument is an infinite value.'
```

File: content/cpp/concepts/math-functions/terms/isinf/isinf.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+  - 'Arithmetic'
+  - 'Functions'
+  - 'Methods'
+  - 'Numbers'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The **`isinf()`** function returns a value telling whether the argument has an infinite value.
```
Comment (SSwiniarski) [inline content/cpp/concepts/math-functions/terms/isinf/isinf.md:16]:
```suggestion
The **`isinf()`** function returns a value indicating whether the argument has an infinite value.
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@ddiegow I did a first review and left a couple of comments.

**Comment by ddiegow** [comment]
Thanks for letting me know @SSwiniarski , didn't realize I was still on that branch when I branched of for this task. I'll be more careful next time, but happy I learnt something new :)

**Review by SSwiniarski — APPROVED** [review]
@ddiegow, Looks good for a second review.

**Comment by Dusch4593** [comment]
@ddiegow Taking a look now for review 2️⃣  😄

File: content/cpp/concepts/math-functions/terms/isinf/isinf.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+  }
+  else {
+    std::cout << "Is infinite" << "\n";
+  }
+  if (result2 == 0) {
+    std::cout << "Not infinite" << "\n";
+  }
+  else {
+    std::cout << "Is infinite" << "\n";
+  }
```
Comment (Dusch4593) [inline content/cpp/concepts/math-functions/terms/isinf/isinf.md:76]:
```suggestion
  if (result == 0) {
    std::cout << "Result: Not infinite" << "\n";
  }
  else {
    std::cout << "Result: Is infinite" << "\n";
  }
  if (result2 == 0) {
    std::cout << "Result2: Not infinite" << "\n";
  }
  else {
    std::cout << "Result2: Is infinite" << "\n";
  }
```

File: content/cpp/concepts/math-functions/terms/isinf/isinf.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+```codebyte/cpp
+#include <iostream>
+#include <cmath>
+
+int main() {
+  int result, result2;
+
+  result = std::isinf(1.0/2);
+  result2 = std::isinf(1.0/0);
```
Comment (Dusch4593) [inline content/cpp/concepts/math-functions/terms/isinf/isinf.md:63]:
```suggestion
  result = std::isinf(1.0/2.0);
  result2 = std::isinf(1.0/0.0);
```

It seems that `result2` causes the program to crash unless the denominator is a floating-point value also.

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@ddiegow Left you some comments/suggestions 😄

File: content/cpp/concepts/math-functions/terms/isinf/isinf.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+  if (result == 0) {
+    std::cout << "Not infinite" << "\n";
+  }
+  else {
+    std::cout << "Is infinite" << "\n";
+  }
+  // Output: Not infinite
+}
+```
+
```
[the remark this answers is not in the document]
Comment (yangc95) [reply content/cpp/concepts/math-functions/terms/isinf/isinf.md:49]: @ddiegow Let's implement this.

**Review by yangc95 — APPROVED** [review]
Looks good to merge! I made a change here: https://github.com/Codecademy/docs/pull/1505/commits/e16f34b847cf7625257ff512a7df6d67c0abde75

**Comment by yangc95** [comment]
@ddiegow TY for your patience with this one! Here's the live link: https://www.codecademy.com/resources/docs/cpp/math-functions/isinf

Let us know if you'd like to become a public contributor so you don't appear an "anonymous contributor."
Screen Shot 2023-02-03 at 6 54 00 AM

cc: @SSwiniarski @KTom101

**Pull request merged**